introduction.md

```diff
@@ -0,0 +1,2 @@
+  Hello, I'm Samuel Pearl. I'm taking the Code School "Mastering Github course".
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.